src/lib/Components/Bidding/Screens/__tests__/BidFlow-tests.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import "react-native"
+import * as renderer from "react-test-renderer"
+
+import { BidFlow } from "../BidFlow"
+
+it("renders properly", () => {
+  const bg = renderer.create(<BidFlow saleArtworkID="5aada729139b216c0bf18103" />).toJSON()
+  expect(bg).toMatchSnapshot()
```

Comment:
Okay! So now that the `BidFlow` component is going to _use_ the `SelectMaxBid` component, this poses a challenge with #1014. We'll need to modify the tests here to have an initial GraphQL state once that PR is merged. I'm happy to take care of that, or pair. 

/cc @sepans